index.html

```diff
@@ -87,7 +87,7 @@
 
 							<li class="nav-item"><a class="nav-link" href="/">Home</a></li>
 							<li class="nav-item"><a class="nav-link" href="about.html">About</a></li>
-							<li class="nav-item"><a class="nav-link" href="speakers.html">Speakers</a></li>
+							<li class="nav-item"><a class="nav-link" href="#">Speakers</a></li>
```

Comment:
@MidigoF  you need to comment this out instead, both this and schedule,
then so so in the about.html, page too

Reply:
doing so